### PR TITLE
misc. typo fixes

### PR DESCRIPTION
**Pull request by luzpaz**
some doxy and some are non-user-facing
(I think i was able to filter out any 3rd party libs...apologies if I included any)

Could not automatically create an issue. Please create one on https://tracker.phpbb.com/ and replace this text with a link to it.

### CODE REVIEW

File: phpBB/phpbb/storage/storage.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ public function put_contents($path, $content)
 	 *
 	 * @param string	$path	The file to read
 	 *
-	 * @throws \phpbb\storage\exception\exception	When the file dont exists
+	 * @throws \phpbb\storage\exception\exception	When the file doesn't exists
```
Comment (Skouat) [inline phpBB/phpbb/storage/storage.php]: `exist` without `s`

**Comment by CHItA** [comment]
This PR needs to be rebased.

**Comment by luzpaz** [comment]
@CHItA rebased

**Comment by marc1706** [comment]
Thanks for your pull request. I have taken the liberty to fix your commit message to adhere to our commit message format and created a new pull request for that:
https://github.com/phpbb/phpbb/pull/5020

**Pull request closed without merging**